app/main.py

```diff
@@ -1,3 +1,67 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> int:
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        else:
+            return Distance(km=self.km + other)
+
+    def __iadd__(self, other: int) -> int:
```

Comment:
The `__iadd__` method should return an instance of the `Distance` class, not an integer. Please correct the return type hint.